test_app/scripts/bootstrap.sh

```diff
@@ -31,12 +38,12 @@ for i in $(seq 1 $MAX_ATTEMPTS); do
     sleep 1
 done
 
-python3 manage.py migrate
-
-DJANGO_SUPERUSER_PASSWORD=password DJANGO_SUPERUSER_USERNAME=admin DJANGO_SUPERUSER_EMAIL=admin@stuff.invalid python3 manage.py createsuperuser --noinput
-
-python3 manage.py authenticators --initialize
-
-python3 manage.py create_demo_data
+if [ -z "$docker_running" ]
```

Comment:
Can we standardize on how we're doing conditionals?

Both syntactically (double vs single brackets, `then` on same line vs next line), and also semantically (I don't get why this is one so different from the above one, it seems to be wanting to do the same check? Can we simplify the above one to be like this one?)

Reply:
I thought the single versus double brackets were meaningful? I'll change the if; then, because I'm sure that doesn't make a difference.

I Google for how to do something and use code from there... I never really got the hang of bash myself and sometimes the syntax does seem like voodoo.